library/button/button.css

```diff
@@ -0,0 +1,99 @@
+.btn {
+  padding: 0.5rem;
+  margin: 0.5rem;
```

Comment:
use px

Reply:
Using font-size as 16px, 0.5 for me is 8px.